static/src/stylesheets/module/facia-garnett/_pillars.scss

```diff
@@ -371,13 +373,8 @@ $pillars: (
         }
     }
 
-    &:not(.fc-item--type-comment) {
-        .fc-item__container.u-faux-block-link--hover {
-            background-color: darken($special-report-dark, 2%);
-        }
-        .fc-item__byline {
-            color: $highlight-main;
-        }
+    .fc-item__container.u-faux-block-link--hover {
+        background-color: darken($special-report-dark, 2%);
```

Comment:
This is returning things back to how they were previously, before any of our fixes on Friday.